README.md

```diff
@@ -10,57 +9,86 @@ What Currently Works
 - Main port screen
 - Resources things (furniture, sound, etc)
 
+What Doesn't Work
+--------------------
+- Anything not listed above.
 
 FAQ
 ---
 
-### Q: Will I be able to import my data from an official server?
+### Q: Will I be able to import my data from an official DMM server?
+Theoretically, yes.   
+The only downside is it will require your API key to import your ships2 data and your admiral information. This may get your account banned for whatever lame DMM reason.  
+It is recommended you start fresh on your copy of this server. 
 
-Theoretically, yes. The only downside is it will require your API key to import your ships2 data and your admiral information. This may get your account banned.
-
-### Q: Why is it built on Python?
+### Q: What is the recommended requirements?
+- Debian or Ubuntu based distro. Fedora and friends may work, but please, YMMV.
+- Python 3.x and knowledge about virtualenv (setup and activate).
+- Internet connection to download the DMM assets.
 
-There aren't actually that many things you can build a web application on, and most of them were out from the start.
+Protip: Use a VM (VMWare, VirtualBox). If you mess up, simply hose it and start squeaky clean. Or you can use an ARM based linux box (R-Pi, ODROID, etc).
 
-* PHP is [an ungodly clusterfuck](http://eev.ee/blog/2012/04/09/php-a-fractal-of-bad-design/) that's insanely unwieldly, performs about as well as a football team in Antarctica and is a nightmare to maintain.  
-  I've written large applications in PHP before, and I'm not going there again.
-
-* Ruby and Java are both massive resource hogs, and this server needs to be able to scale well, preferably without me having me hire another server I can't pay for.  
-  I run a JIRA (Java) and a Discourse (Ruby) instance, and they both chew up nearly everything their servers have to give.
+### Q: Why is it built on Python?
+There aren't actually that many things you can build a web application on, and most of them were out from the start.  
+There was a large rant by one the kcsrv owner, I (SoftwareGuy) who improved this FAQ have condensed it down to make it less ranty.
 
+* PHP is [an ungodly clusterfuck](http://eev.ee/blog/2012/04/09/php-a-fractal-of-bad-design/).
+* Ruby and Java are both massive resource hogs, and this server needs to be able to scale well. Ruby / Java seem to eat the server alive.
```

Comment:
"Ruby and Java", not "Ruby/Java" - the latter implies that they're interchangeable, which they're most certainly not…